### PR TITLE
fix: correct rag tool description

### PR DESCRIPTION
**Pull request by yuye-aws**
### Description

Change RAG tool description to **Use this tool to retrieve helpful information to answer question.**
 
### Issues Resolved

Correct RAG tool description.
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/RAGTool.java [hunk 1]
```diff
@@ -54,7 +54,7 @@
 @ToolAnnotation(RAGTool.TYPE)
 public class RAGTool extends AbstractRetrieverTool {
     public static final String TYPE = "RAGTool";
-    private static String DEFAULT_DESCRIPTION = "Use this tool to run any model.";
+    private static String DEFAULT_DESCRIPTION = "Use this tool to retrieve helpful information to answer question.";
```
Comment (dhrubo-os) [inline ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/RAGTool.java:57]: retrieve helpful information `from where`?

Comment (ylwu-amzn) [reply ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/RAGTool.java:57]: This tool is different with VectorDBTool which is to retrieve helpful information. This RAGTool is to help answer questions with user configured knowledge base which provide helpful information.

Comment (mingshl) [reply ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/RAGTool.java:57]: Based on the defination of [RAG](https://aws.amazon.com/what-is/retrieval-augmented-generation/#:~:text=Augmented%20Generation%20requirements%3F-,What%20is%20Retrieval%2DAugmented%20Generation%3F,sources%20before%20generating%20a%20response.), I will suggest we could describe RAG tool as to `optimize the output of a large language model to answer questions`.

Comment (yuye-aws) [reply ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/RAGTool.java:57]: Agreed. Will change the description.

**Review by ylwu-amzn — APPROVED** [review]

**Pull request merged**